src/main/java/com/edmunds/tools/databricks/maven/JobMojoNoProject.java

```diff
@@ -45,6 +45,12 @@
 
     @Override
     protected JobTemplateModel getJobTemplateModel() throws MojoExecutionException {
-        return JobTemplateModel.loadJobTemplateModelFromFile(jobTemplateModelFile);
+        JobTemplateModel serializedJobTemplate = JobTemplateModel.loadJobTemplateModelFromFile(jobTemplateModelFile);
+        //We now set properties that are based on runtime and not buildtime. Ideally this would be enforced.
+        //I consider this code ugly
```

Comment:
Why is it ugly? Is there a better way then?

Reply:
It's same as 'UpsertJobMojoNoProject' 
The problem is that 'job-template-model.json' file is generated on bulid-time based on  pom.xml  
.....
 <plugins>
  <plugin>
    <configuration>
                        <databricksRepo>XXX</databricksRepo>
                        <environment>QA</environment>
                        <host>...
      </configuration>
    </plugin>
 </plugins>
....

But environment should be used by runtime.

Any idea?